cassandra-adapter/src/main/java/ch/unibas/dmi/dbis/polyphenydb/adapter/cassandra/CassandraStore.java

```diff
@@ -0,0 +1,321 @@
+        // TODO: Implement
+        log.warn( "updateColumnType is not implemented yet." );
+    }
+
+
+    @Override
+    public String getAdapterName() {
+        return ADAPTER_NAME;
+    }
+
+
+    @Override
+    public List<AdapterSetting> getAvailableSettings() {
+        return AVAILABLE_SETTINGS;
+    }
+
+
+    @Override
+    public void shutdown() {
+        // TODO: Implement
+        try {
+            this.session.close();
+        } catch ( RuntimeException e ) {
+            log.warn( "Exception while shutting down " + getUniqueName(), e );
+        }
+//        log.warn( "shutdown is not implemented yet." );
+    }
+
+
+    @Override
+    protected void reloadSettings( List<String> updatedSettings ) {
+        // TODO: Implement
+        log.warn( "reloadSettings is not implemented yet." );
+    }
+
+
+    private DataType getDataType( PolySqlType polySqlType ) {
+        switch ( polySqlType ) {
+            case BOOLEAN:
+                return DataTypes.BOOLEAN;
+            case VARBINARY:
+                throw new RuntimeException( "Unsupported datatype: " + polySqlType.name() );
+            case INTEGER:
+                return DataTypes.INT;
+            case BIGINT:
+                return DataTypes.BIGINT;
+            case REAL:
+                throw new RuntimeException( "Unsupported datatype: " + polySqlType.name() );
+            case DOUBLE:
+                return DataTypes.DOUBLE;
+            case DECIMAL:
+                return DataTypes.DECIMAL;
+            case VARCHAR:
+                return DataTypes.TEXT;
+            case TEXT:
+                return DataTypes.TEXT;
+            case DATE:
+                return DataTypes.DATE;
+            case TIME:
+                return DataTypes.TIME;
+            case TIMESTAMP:
+                return DataTypes.TIMESTAMP;
+        }
+        throw new RuntimeException( "Unknown type: " + polySqlType.name() );
+    }
+
+
+    private String getTypeString( PolySqlType polySqlType ) {
+        switch ( polySqlType ) {
+            case BOOLEAN:
+                return "BOOLEAN";
+            case VARBINARY:
+                throw new RuntimeException( "Unsupported datatype: " + polySqlType.name() );
+            case INTEGER:
+                return "INT";
+            case BIGINT:
+                return "BIGINT";
+            case REAL:
+                throw new RuntimeException( "Unsupported datatype: " + polySqlType.name() );
+            case DOUBLE:
+                return "DOUBLE";
+            case DECIMAL:
+                return "DECIMAL";
+            case VARCHAR:
+                return "VARCHAR";
+            case TEXT:
+                return "TEXT";
+            case DATE:
+                return "DATE";
+            case TIME:
+                return "TIME";
+            case TIMESTAMP:
+                return "TIMESTAMP";
+        }
+        throw new RuntimeException( "Unknown type: " + polySqlType.name() );
+    }
```

Comment:
> Thx for this PR. I add some comments and questions. It would also be nice to have some basic integration tests for the new adapter.

Any existing integration tests that could be used as a basis/inspiration on how to do it?

Reply:
Have a look at `dbms/src/test/java/ch/unibas/dmi/dbis/polyphenydb/jdbc/JdbcMetaTest.java`

Reply:
Addressed in 97b5d3deb6440746c45c8e9e80826c388d989ab3